types/serialization.go

```diff
@@ -182,6 +183,13 @@ func (c *Commit) FromProto(other *pb.Commit) error {
 	return nil
 }
 
+// Uint64ToByteSlice converts a uint64 into an 8 byte little endian slice
+func Uint64ToByteSlice(v uint64) []byte {
```

Comment:
As this function is used in a single place (`da/mock`), and this place is mostly "for test purposes", I would keep this as unexported function inside `da/mock` package (similarly to [store package](https://github.com/celestiaorg/optimint/blob/main/store/store.go#L169)).
Personally, I prefer function names to tell "what" func does, "why" we have it/use it.

Reply:
I've moved this to da/mock, renamed to match the function in the store package (though with no prefix), and switched to BigEndian.